back/node_watcher/src/nomidot.spec.ts

```diff
@@ -16,14 +15,15 @@ describe('Nomidot Tasks', () => {
   beforeEach(
     async (done): Promise<void> => {
       api = await ApiPromise.create({ provider: new WsProvider(parityKusama) });
+
       done();
     }
   );
 
   it('should read a single task', async done => {
-    const intentions = await api.query.staking.validators();
+    const currentValidators = await api.query.session.validators();
```

Comment:
staking.validators is a linked map, was taking more than 5 seconds and test failed after that.

Reply:
ah, sweet, thank you